Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -191,9 +191,18 @@ private IEnumerable<VideoInfo> GetVideosGroupedByVersion(List<VideoInfo> videos)
 
                 list.Add(ordered[0]);
 
-                list[0].AlternateVersions = ordered.Skip(1).Select(i => i.Files[0]).ToList();
+                var alternateVersionsLen = ordered.Count - 1;
+                var alternateVersions = new VideoFileInfo[alternateVersionsLen];
+                for (int i = 0; i < alternateVersionsLen; i++)
+                {
+                    alternateVersions[i] = ordered[i + 1].Files[0];
+                }
+
+                list[0].AlternateVersions = alternateVersions;
```

Comment:
original looked easier to read, is performance gain here that much?

Reply:
Depends on the amount of alternate versions. The largest speed up is that there’s only 1 allocation now instead of a growing list.